client/pages/chat-room.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import Header from '../components/header';
+
+export default class Chatroom extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      message: ''
+    };
+    this.sendMessage = this.sendMessage.bind(this);
+    this.typeMessage = this.typeMessage.bind(this);
+  }
+
+  typeMessage(event) {
+    this.setState({
+      message: event.target.value
+    });
+  }
+
+  sendMessage(event) {
+    event.preventDefault();
+    const { sender, receiver } = this.props;
+    const message = {
+      content: this.state.message,
+      sender,
+      receiver
```

Comment:
you can remove/ignore the sender for now. the sender will always be determined on the server side. the client is never allowed to have authority and declare who they are.